omim2obo/main.py

```diff
@@ -280,49 +280,72 @@ def omim2obo(use_cache: bool = False):
             graph.add((OMIM[mim_number], RDFS.subClassOf, OMIMPS[ps_id]))
 
     # Morbid map
-    morbid_map: Dict = parse_morbid_map(get_mim_file('morbidmap', download_files_tf))
-    for mim_number, mim_data in morbid_map.items():
-        # todo?: unused `mim_data` keys. Should they be used?
-        #  - phenotype_label: Similar to p_lab in 'assocs', but has more info
-        #  - gene_symbols
-        # - add cyto location
-        cyto_location: str = mim_data['cyto_location']
-        if cyto_location:
-            # What's 9606chr - joeflack4 2021/11/11
-            chr_id = '9606chr' + cyto_location
-            # RO:0002525 (is subsequence of)
-            # https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0002525
-            add_subclassof_restriction(graph, RO['0002525'], CHR[chr_id], OMIM[mim_number])
-        assocs: List[Dict] = mim_data['phenotype_associations']
-        # - add associations
+    gene_phenotypes: Dict[str, Dict] = parse_morbid_map(get_mim_file('morbidmap', download_files_tf))
+
+    # Gene-Chromosome relationships
+    # - Cyto location: Add RO:0002525 (is subsequence of)
+    # https://www.ebi.ac.uk/ols/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0002525
+    for gene_mim, gene_data in gene_phenotypes.items():
+        if gene_data['cyto_location']:
+            chr_id = '9606chr' + gene_data['cyto_location']  # todo: document meaning of 9606chr
```

Comment:
9606 is the ncbi taxonomy identifier for Homo sapiens

Reply:
thank you!
i'll add that